resources/config.rb

```diff
@@ -0,0 +1,73 @@
+property :name, String, name_property: true
+property :home_net, String,     default: 'any'
+property :external_net, String, default: 'any'
```

Comment:
I wonder if we want to continue kicking this can down the road. This is certainly better than the previous solution, but still requires us to add every single config option as a property